lib/DxilPIXPasses/DxilDbgValueToDbgDeclare.cpp

```diff
@@ -497,6 +775,8 @@ static llvm::DIType *DITypePeelTypeAlias(
     case llvm::dwarf::DW_TAG_const_type:
     case llvm::dwarf::DW_TAG_typedef:
     case llvm::dwarf::DW_TAG_pointer_type:
+      return DITypePeelTypeAlias(
+          DerivedTy->getBaseType().resolve(EmptyMap));
     case llvm::dwarf::DW_TAG_member:
```

Comment:
In a couple of places I've added an apparently (well, actually) unnecessary extra case for DW_TAG_member. This is a quality-of-life thing: it's much easier to set a breakpoint on members like this.